operate/services/manage.py

```diff
@@ -1491,19 +1491,25 @@ def fund_service_single_chain(  # pylint: disable=too-many-arguments,too-many-lo
                     address=key.address,
                 )
                 self.logger.info(
-                    f"Agent {key.address} Asset: {asset_address} balance: {agent_balance}"
+                    f"[FUNDING_JOB] Agent {key.address} Asset: {asset_address} balance: {agent_balance}"
                 )
                 if agent_fund_threshold > 0:
-                    self.logger.info(f"Required balance: {agent_fund_threshold}")
+                    self.logger.info(f"[FUNDING_JOB] Required balance: {agent_fund_threshold}")
                     if agent_balance < agent_fund_threshold:
```

Comment:
Why is this check not performed in safe as well?

Reply:
it's done on line 1545